src/main/steps/common/webpack/js.njk

```diff
@@ -0,0 +1 @@
+<script src="main.56073a5010f6b27fe707.js"></script>
```

Comment:
there one change i think i forgot to push is, in the js-template file, adding the / before the main part. otherwise these scripts wont work.

Reply:
cant seem to suggest change as the file didnt change.

Reply:
changed it